feg/cloud/go/services/feg_relay/servicers/feg_to_gw_relay.go

```diff
@@ -140,7 +140,29 @@ func getFegServedIds(networkId string) ([]string, error) {
 	if !ok || networkFegConfigs == nil {
 		return []string{}, fmt.Errorf("invalid federation network config found for network: %s", networkId)
 	}
-	return networkFegConfigs.ServedNetworkIds, nil
+	// If this is a NH FeG Network, add served networks from ServedNhIds FeG networks, if not - return ServedNhIds
```

Comment:
isnt it possible to have a nh network and still have some gateways direclty connected?

Wouldnt it be better append the results? Meaning if we have nh return the configured local networks and the nh networks

Reply:
yea, that's what it does - takes local Served Network IDs (nids := networkFegConfigs.ServedNetworkIds) & appends Served Network IDs from NH FeGs (nids = append(nids, nhNetworkFegConfigs.ServedNetworkIds...)